Rakefile

```diff
@@ -6,7 +6,6 @@ WORKSPACE = 'StencilSwiftKit'
 TARGET_NAME = 'Tests'
```

Comment:
Should we document somewhere (comment at the top of each `rakelib/*.rake` file? Elsewhere?) what global variables are used by various tasks, and thus which variables are expected to be defined in each repo's Rakefile?